apps/src/StudioApp.js

```diff
@@ -2111,8 +2109,8 @@ StudioApp.prototype.handleEditCode_ = function (config) {
             !this.editor.paletteEnabled ? 'none' : 'inline-block';
         this.resizeToolboxHeader();
       }
-    }).bind(this);
-    dom.addClickTouchEvent(hideToolboxHeader, handleTogglePalette);
+    };
+    dom.addClickTouchEvent(hideToolboxIcon, handleTogglePalette);
```

Comment:
We now only hide the header if you click the chevron icon, not anywhere on the header - this allows us to add a second interactable element to the header.